### ISSUE TITLE
Active Item on Asset Page Breadcrumbs has Insufficient Color Contrast

### ISSUE THREAD

**Issue by jmcglone** [mlibrary/heliotrope#618]
ACTIVE BREADCRUMB: Element has insufficient color contrast of 4.11 (foreground color: #777777, background color: #f5f5f5, font size: 10.5pt, font weight: normal)

`#main > div:nth-of-type(1) > .col-md-12.main-header > div:nth-of-type(1) > .breadcrumbs.col-sm-12 > .breadcrumb > .active`
`<li class="active">E. T. A. Hoffmann, self-portrait</li>`

- [ ] 1.4.3 Contrast (Minimum): The visual presentation of text and images of text has a contrast ratio of at least 4.5:1, except for the following: (Level AA)
Large Text: Large-scale text and images of large-scale text have a contrast ratio of at least 3:1;
Incidental: Text or images of text that are part of an inactive user interface component, that are pure decoration, that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement.
Logotypes: Text that is part of a logo or brand name has no minimum contrast requirement.
